### PR TITLE
MODSOURMAN-281 - Ignore event if there are no eligible profiles

### PR DESCRIPTION
**Pull request by RuslanLavrov**

### CODE REVIEW

File: src/main/java/org/folio/processing/events/EventManager.java [hunk 1]
```diff
@@ -15,27 +18,30 @@
...
-import io.vertx.core.logging.Logger;
-import io.vertx.core.logging.LoggerFactory;
-
 /**
  * The central class to use for handlers registration and event handling.
  */
 public final class EventManager {
 
+  public static final String POST_PROCESSING_INDICATOR = "POST-PROC";
+  public static final String POST_PROCESSING_RESULT_EVENT_KEY = "POST-PROCESSING_RESULT_EVENT";
```
Comment (KaterynaSenchenko) [inline src/main/java/org/folio/processing/events/EventManager.java:37]:
```suggestion
  public static final String POST_PROCESSING_RESULT_EVENT_KEY = "POST_PROCESSING_RESULT_EVENT";
```

File: src/main/java/org/folio/processing/events/EventManager.java [hunk 2]
```diff
@@ -15,27 +18,30 @@
...
 
-import io.vertx.core.logging.Logger;
-import io.vertx.core.logging.LoggerFactory;
-
 /**
  * The central class to use for handlers registration and event handling.
  */
 public final class EventManager {
 
+  public static final String POST_PROCESSING_INDICATOR = "POST-PROC";
```
Comment (KaterynaSenchenko) [inline src/main/java/org/folio/processing/events/EventManager.java:36]:
```suggestion
  public static final String POST_PROCESSING_INDICATOR = "POST_PROCESSING";
```

File: src/test/java/org/folio/processing/events/handlers/InstanceEventHandlerPostProcessor.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import org.folio.processing.events.services.handler.EventHandler;
+
+import java.util.concurrent.CompletableFuture;
+
+import static org.folio.ActionProfile.Action.UPDATE;
+import static org.folio.ActionProfile.FolioRecord.INSTANCE;
+import static org.folio.DataImportEventTypes.DI_INVENTORY_INSTANCE_UPDATED;
+import static org.folio.rest.jaxrs.model.ProfileSnapshotWrapper.ContentType.ACTION_PROFILE;
+
+public class InstanceEventHandlerPostProcessor implements EventHandler {
```
Comment (KaterynaSenchenko) [inline src/test/java/org/folio/processing/events/handlers/InstanceEventHandlerPostProcessor.java:15]:
```suggestion
public class InstancePostProcessingEventHandler implements EventHandler {
```

File: src/test/java/org/folio/processing/events/handlers/UpdateInstanceEventHandler.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import org.folio.processing.events.services.handler.EventHandler;
+
+import java.util.concurrent.CompletableFuture;
+
+import static org.folio.ActionProfile.Action.UPDATE;
+import static org.folio.ActionProfile.FolioRecord.INSTANCE;
+import static org.folio.DataImportEventTypes.DI_INVENTORY_INSTANCE_UPDATED;
+import static org.folio.rest.jaxrs.model.ProfileSnapshotWrapper.ContentType.ACTION_PROFILE;
+
+public class UpdateInstanceEventHandler implements EventHandler {
```
Comment (KaterynaSenchenko) [inline src/test/java/org/folio/processing/events/handlers/UpdateInstanceEventHandler.java:15]:
```suggestion
public class InstanceUpdateEventHandler implements EventHandler {
```

File: src/test/java/org/folio/processing/events/handlers/UpdateInstanceEventHandler.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+  public static final String POST_PROC_INIT_EVENT = "DI_INVENTORY_INSTANCE_UPDATED_READY_FOR_POST_PROCESSING";
```
Comment (KaterynaSenchenko) [inline src/test/java/org/folio/processing/events/handlers/UpdateInstanceEventHandler.java:17]: Please do not cut off the word PROCESSING to PROC, it looks weird and is a bit confusing

**Review by KaterynaSenchenko — APPROVED** [review]

File: src/test/java/org/folio/processing/events/handlers/UpdateInstanceEventHandler.java (hunk 4 above)
Comment (RuslanLavrov) [reply src/test/java/org/folio/processing/events/handlers/UpdateInstanceEventHandler.java:15]: but it corresponds to the existing naming format of handlers 
https://github.com/folio-org/data-import-processing-core/blob/d3da666486a665c464d57d5323af9f73c6b54bd5/src/test/java/org/folio/processing/events/handlers/CreateInstanceEventHandler.java#L15

**Review by VRohach — APPROVED** [review]

**Review by OleksiiKuzminov — APPROVED** [review]

**Pull request merged**